module1.js

```diff
@@ -23,179 +23,106 @@ function question1(name1) {
   }
 }
 
-
 /**
-  +++++++++++++++++++++++++++++++++++++++++++
   Question 2:
-  IF / ELSE Statements:
-  Check if two numbers are equal, if they are return true or otherwise return false
-  @param {number} number1 a regular integer
-  @param {number} number2 a regular integer
-  @returns {boolean} true or false
-*/
+  Write a function that compares two input integers.
 
-// function question2(number1, number2) {
```

Comment:
If we're uncommenting the question functions we might as well uncomment them in `module.exports` too. The tests should still fail (as all of the functions are no-ops)

Reply:
Done - I removed the instruction to uncomment the module.exports as well